### DISCUSSION TITLE
Error message in strategy.py

### DISCUSSION THREAD

**Discussion by hyperstrings2003** [fja05680/pinkfish#69] [General]
I converted all examples from *.ipynb to *.py with  python - jupyter nbconvert --to script */*.ipynb

and tried to run pinkfish-master\examples\080.double-7s\strategy.py

and always got error message

 File "pinkfish-master\examples\080.double-7s\strategy.py", line 19, in <module>
    import strategy
  File "pinkfish-master\examples\080.double-7s\strategy.py", line 92, in <module>
    s = strategy.Strategy(symbol, capital, start, end, options)
AttributeError: partially initialized module 'strategy' has no attribute 'Strategy' (most likely due to a circular import). Did you mean: 'strategy'?

I tried on 3 different computers: Windows, RHEL 9.0 and Ubuntu 22.10 with the same error message

also I was only able to pip install pinkfish 1.15.0 and not 1.16.0 where TA-LIb pip installation crashed complaining "no Python.h" (again on all 3 operating systems)

What am I missing?

**Comment by fja05680** [comment]
It appears that install via pip is currently broken.  Also the instructions for developer mode install were out of sequence.  I just corrected that:
https://fja05680.github.io/pinkfish/

I was able to get everything working following the updated instructions at the above link using a Ubuntu 22.04 VM.  Please give that a try.  I'll fix pip install on the next version release, probably within the next couple of weeks.
